ccw.core/src/clj/ccw/eclipse.clj

```diff
@@ -620,10 +651,58 @@
         (let [s (f monitor)]
           (if (instance? IStatus s)
             s
-            (Status/OK_STATUS)))
+            Status/OK_STATUS))
         (catch Exception e
           (CCWPlugin/createErrorStatus (format "Unexpected exception while executing Job %s" name), e))))))
 
+(defn cancelable-workspace-job
```

Comment:
I wanted to isolate such a big change...when the test will confirm that everything is fine...well I can just change the name (I know I know, overly defensive programming...)